ios/config.json

```diff
@@ -7,7 +7,10 @@
   "frameworks": [
     "FirebaseAnalytics.framework",
     "FirebaseCore.framework",
+    "FirebaseCoreDiagnostics.framework",
+    "FirebaseNanoPB.framework",
```

Comment:
do we need these frameworks ?

Reply:
Firebasecore depends on FirebaseCoreDiagnostics and FirebaseNanoPB